.github/workflows/release.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
         run: sam build --template-file firehose-template.yaml --region us-east-2
 
       - name: Package SAM Application
-        run: sam package --s3-bucket unified-lambda-serverless-1 --output-template-file firehose-template.yaml --region us-east-2
+        run: sam package --s3-bucket unified-lambda-serverless --output-template-file firehose-template.yaml --region us-east-2
```

Comment:
I am not seeing any code giving any explicit role/permission for Jim Ruppert's account. How will it work ? For Logging Integration it was working because the credentials were stored in the repo.

Reply:
The secret is already present in github as AWS_SAR_PUBLISH_ROLE. I just modified this secret to add new Jim Ruppert IAM role arn. I'll rename this to AWS_S3_PUBLISH_ROLE